docs/release-notes.md

```diff
@@ -3,6 +3,32 @@
 - Table of Contents
 {:toc}
 
+## Version 1.9.0
+
+The 1.9.0 release of MCUboot contains various bug fixes, improves
+support on some recent targets, and adds support for devices with a
+write alignment larger than 8.
+
+This change introduces a potentially incompatible change to the format
```

Comment:
I would mention that new feature is accessible over `MCUBOOT_BOOT_MAX_ALIGN`, if this is undefined or defined to 8 the mcuboot is backward compatible.